lib/internal/util/inspect.js

```diff
@@ -528,7 +531,7 @@ function strEscape(str) {
           continue;
         }
       }
-      result += `${StringPrototypeSlice(str, last, i)}${`\\u${point.toString(16)}`}`;
+      result += `${StringPrototypeSlice(str, last, i)}\\u${point.toString(16)}`;
```

Comment:
```suggestion
      result += `${StringPrototypeSlice(str, last, i)}\\u${NumberPrototypeToString(point, 16)}`;
```

Reply:
Since it's not an error case, I would rather not change these.